frontends/web/src/hooks/api.ts

```diff
@@ -76,7 +76,7 @@ export const useSync = <T>(
 ): (T | undefined) => {
     const [respose, setResponse] = useState<T>();
     const mounted = useMountedRef();
-    const onData = (data) => {
+    const onData = (data: T) => {
```

Comment:
is this typing here required? i would expect that it is inferred, as it is already typed above with `apiCall: () => Promise<T>.

can't hurt of course.

Reply:
I am not sure why exactly, TypeScript complained for `useSync` didn't complain for `useLoad` and `useSubscribe` and correctly says data is `T`